util/generate-ami-list.py

```diff
@@ -131,12 +131,26 @@ def get_images_ec2(filters, owner, region_name):
     try:
         ec2 = boto3.client("ec2", region_name=region_name)
         images = ec2.describe_images(Owners=[owner], Filters=filters)
-        return images
+        return latest_image_per_os(images)
     except ClientError:
         print("Warning: non authorized in region '{0}', skipping".format(region_name))
         pass
 
 
+def latest_image_per_os(images):
+    # filter for the latest image per OS
+    images_filtered = {"Images": []}
+    for key, value in distros.items():
+        ami_filtered_and_sorted = sorted(
+            filter(lambda ami: ("-" + value + "-") in ami["Name"], images["Images"]),
```

Comment:
Minor: If this code is only executed by python3 you can write: 
```python
f"-{value}-"
```
or in any case you can use
```python
"-{0}-".format(value)"
```

Reply:
thanks